connect/eaas/runner/main.py

```diff
@@ -157,6 +182,8 @@ def start(data):
     if data.unsecure:
         logger.warning('Websocket connections will be established using unsecure protocol (ws).')
 
+    check_runner_version()
```

Comment:
This check should be called in main function just after the logger configuration and before the start function is called.
If the version is not the latest the execution must be stopped.
We should add a flag to the Argument Parser like --skip-version-check so we can allow developers to run the runner event if the version is not the latest.